### PR TITLE
feat(menu): add setup helper

### PR DESCRIPTION
**Pull request by virror**
Add a new menu item to help with setting up a new Intractable Object.

### CODE REVIEW

File: Assets/VRTK/Editor/VRTK_ObjectSetup.cs [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+        [MenuItem("Window/VRTK/Setup Interactable Object")]
+        private static void Init()
+        {
+            VRTK_ObjectSetup window = (VRTK_ObjectSetup)EditorWindow.GetWindow(typeof(VRTK_ObjectSetup));
+            
+            window.minSize = new Vector2( 300f, 300f );
+            window.maxSize = new Vector2( 400f, 300f );
+            
+            window.autoRepaintOnSceneChange = true;
+            window.titleContent.text = "Setup Object";
```
Comment (thestonefox) [inline Assets/VRTK/Editor/VRTK_ObjectSetup.cs]: worth naming it "Setup Interactable Object" to make it more obvious what the tab is?

File: Assets/VRTK/Editor/VRTK_ObjectSetup.cs [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+        private bool holdGrab = false;
+        private bool useUse = false;
+        private bool useIfGrabbed = false;
+        private bool holdUse = false;
+        private PrimaryGrab primGrab;
+        private SecondaryGrab secGrab;
+        private bool disableIdle = true;
+        private bool addrb = true;
+
+        [MenuItem("Window/VRTK/Setup Interactable Object")]
```
Comment (thestonefox) [inline Assets/VRTK/Editor/VRTK_ObjectSetup.cs]: I quite like the idea of having this as a main menu option

"VRTK/Setup Interactable Object"

thoughts?

**Comment by virror** [comment]
I have some other small things i want to change, so i will move the menu to main instead.
 "Setup Interactable Object" dont fit the small Unity tabs, that's why i used a shorter name instead.

**Comment by getwilde** [comment]
For this usage, "Set up" should be two words.  But grammer police are annoying, so you can just ignore me.   :)

File: Assets/VRTK/Editor/VRTK_ObjectSetup.cs (hunk 1 above)
Comment (getwilde) [inline Assets/VRTK/Editor/VRTK_ObjectSetup.cs]: For this usage, "Set up" should be two words. But grammer police are annoying, so you can just ignore me. :)

**Comment by virror** [comment]
Ok, this is ready to be merged now.

**Pull request merged**